### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by adrianacostep33**

### CODE REVIEW

File: frontend/src/components/movie-card-content/MovieCardContent.styled.tsx [hunk 1]
```diff
@@ -65,7 +66,7 @@ export const Rating = styled.div`
 `;
 
 export const Score = styled.div`
```
Comment (roxanap33) [inline frontend/src/components/movie-card-content/MovieCardContent.styled.tsx]: Please delete this component if it is not used

File: frontend/src/components/button/Button.styled.tsx [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+  padding: ${rem(12)};
   border-radius: ${rem(10)};
-  color: ${(props) =>
-    props.type === "text" ? `${pallete.PLATINUM}` : `${pallete.SPACE_CADET}`};
-  font-size: ${rem(16)};
+  font-size: ${rem(10)};
+  background-color: ${(props) =>
+    props.type === "primary" && `${pallete.PLATINUM} `};
+  background-color: ${(props) => props.type === "secondary" && `transparent`};
+
```
Comment (ioanagirbap33) [inline frontend/src/components/button/Button.styled.tsx:17]: Why is there an empty line?

File: frontend/src/components/movie-card-content/MovieCardContent.tsx [hunk 3]
```diff
@@ -40,7 +40,6 @@ function MovieCardContent({
         <MovieDetailsRight>
           <Rating>
             <StarIcon sx={{ color: FRENCH_MAUVE }} />
-
             <Score>4.5</Score>
```
Comment (roxanap33) [inline frontend/src/components/movie-card-content/MovieCardContent.tsx]: Please use the function for calculating the average rating for every movie

File: frontend/src/components/button/Button.styled.tsx (hunk 2 above)
Comment (adrianacostep33) [reply frontend/src/components/button/Button.styled.tsx:17]: I added this for readability

**Review by ioanagirbap33 — APPROVED** [review]

**Review by roxanap33 — APPROVED** [review]
All good

**Pull request merged**